Lib/idlelib/config.py

```diff
@@ -438,6 +439,10 @@ def GetExtensions(self, active_only=True,
         """
         extns = self.RemoveKeyBindNames(
                 self.GetSectionList('default', 'extensions'))
+        for extn in ['AutoComplete','CodeContext','FormatParagraph','ParenMatch']:
+            extns.remove(extn)
+            # specific exclusions because we are storing config for mainlined old
+            # extensions in config-extensions.def for backward compatibility
```

Comment:
Right code, wrong place.  For me, IDLE startup failed because I have customized AutoComplete.popupwait and therefore have an AutoComplete entry in config-extensions.cfg, which resulted in AutoComplete being added back to extns by the next 4 lines.  Problem fixed by moving exclusions after extns building is complete.